packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -117,7 +117,8 @@ export const COMPOSITE_ONLY_ICONS: CompositeIcons = {
   /* @conditional-compile-remove(teams-meeting-conference) */
   JoinByPhoneWaitToBeAdmittedIcon: <Clock20Filled />,
   PeoplePaneMoreButton: <MoreHorizontal20Filled />,
-  StopAllSpotlightMenuButton: <VideoPersonStarOff20Filled />
+  StopAllSpotlightMenuButton: <VideoPersonStarOff20Filled />,
+  WaitingScreenIcon: <>&#9749;</>
```

Comment:
This would be a one off icon in this style.
Rather than doing something like this and having it as an "icon". 
Should we have a string CONSTANT that can change this to any text/emoji in format of UTF-8